src/qudi/hardware/ni_x_series/ni_x_series_in_streamer.py

```diff
@@ -640,7 +640,7 @@ def _init_digital_tasks(self):
             # Set up digital counting tasks
             for i, chnl in enumerate(digital_channels):
                 chnl_name = f'/{self._device_name}/{chnl}'
-                task_name = f'PeriodCounter_{chnl}'
+                task_name = f'PeriodCounter_{self._device_name}_{chnl}'
```

Comment:
Looks fine as a quick fix.
I guess this could still lead to issues in case there are multiple instances of an `NIXSeriesInStreamer` using the same pfi channel. On the hardware side this should still be allowed when the task is using another clock.

Maybe one could extend it like [here](https://github.com/Ulm-IQO/qudi-iqo-modules/blob/4d6795438a2ed844956cf8ecabebfbe046cc5c2e/src/qudi/hardware/ni_x_series/ni_x_series_in_streamer.py#L573) with `id(self)` or use `self.module_uuid` as well such that the task name is unique for each instance?